GitUI/Avatars/BackupAvatarProvider.cs

```diff
@@ -29,15 +29,33 @@ public async Task<Image> GetAvatarAsync(string email, string name, int imageSize
         {
             try
             {
-                return await _inner.GetAvatarAsync(email, name, imageSize) ?? _backupImage;
+                return await _inner.GetAvatarAsync(email, name, imageSize) ?? BackupImageResized();
             }
             catch
             {
-                return _backupImage;
+                return BackupImageResized();
+            }
+
+            Image BackupImageResized()
```

Comment:
To confirm, this is always invoked on the main thread, right?
Because if it isn't, we'll need to synchronise it.

Reply:
I thought it was because it is the case in the application code. But not in the Gource plugin...
Anyway, fixed.